src/main/kotlin/io/emeraldpay/dshackle/Config.kt

```diff
@@ -61,16 +63,20 @@ open class Config(
     private var configFilePath: File? = null
 
     init {
-        configFilePath = getConfigPath()
-        Global.version = env.getProperty("version.app", Global.version).let {
-            if (it.contains("SNAPSHOT")) {
-                listOfNotNull(it, env.getProperty("version.commit")).joinToString("-")
-            } else {
-                it
+        if (!env.activeProfiles.contains("test")) {
+            configFilePath = getConfigPath()
+            Global.version = env.getProperty("version.app", Global.version).let {
+                if (it.contains("SNAPSHOT")) {
+                    listOfNotNull(it, env.getProperty("version.commit")).joinToString("-")
+                } else {
+                    it
+                }
             }
-        }
 
-        Security.addProvider(BouncyCastleProvider())
+            Security.addProvider(BouncyCastleProvider())
+        } else {
```

Comment:
let's inverse If to have clear understanding
`if( env.activeProfiles.contains("test")) ....`